src/main/resources/idprovider/idprovider.js

```diff
@@ -14,7 +39,20 @@ exports.handle401 = function() {
     };
 };
 
-exports.get = function() {
+exports.get = function(req) {
+    const rawPath = req.rawPath;
+    if (!startsWith(rawPath, '/api/idprovider/')) {
```

Comment:
there is no API in 7.x. This code should not exist in this branch

Reply:
Right, I just copied code from my other branch which was based on master